### PR TITLE
packagegroup-rpb-tests-x11: Add gst-validate and xvfb

### PR DESCRIPTION
**Pull request by alimon**
The gst-validate serves to test gstreamer components and it
needs xvfb to run without display attached.

Signed-off-by: Aníbal Limón <anibal.limon@linaro.org>

### CODE REVIEW

**Comment by fboudra** [comment]
nitpick: keep alphabetical order

**Comment by alimon** [comment]
The gst-validate is integrated into oe-core by now, 

http://git.openembedded.org/openembedded-core/commit/?id=4f61c35e4a

**Comment by ndechesne** [comment]
hmm. i missed to review that one earlier.. sorry.

gst-validate is in oe-core/master, but not rocko. it was brought up into meta-backports/rocko . So if we build with meta-rpb and meta-backports, then this patch is ok. however building without meta-backports will result into an error, which we should avoid.

So only merging such change in meta-rpb/master is ok.

We had a similar situation in the past and we ended up doing that:

https://git.linaro.org/openembedded/meta-backports.git/commit/?h=morty&id=373e2988dd54712231492a8e1344275c8e5345b1

basically using a bbappend for the packagegroup in meta-backports.

**Comment by alimon** [comment]
@ndechesne Yes we have already gst-validate on rocko, i sent the review to backports previous integration in oe-core master, https://git.linaro.org/openembedded/meta-backports.git/commit/?h=rocko&id=204156793f9f6881f6332eef2eea1aa0a11bec9a

**Comment by ndechesne** [comment]
i know. but it's in meta-backports. so anyone builds with meta-rpb and without meta-backports, then it will fail. hence what I suggested.

**Comment by alimon** [comment]
@ndechesne Ok got it, i will make the bbappend.

**Comment by alimon** [comment]
To add bbappend into meta-backports.
